sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/models/ChatChoice.java

```diff
@@ -42,29 +42,15 @@ public final class ChatChoice {
      */
     @Generated
     @JsonProperty(value = "delta")
-    private ChatMessage delta;
-
-    /**
-     * Creates an instance of ChatChoice class.
-     *
-     * @param index the index value to set.
-     * @param finishReason the finishReason value to set.
-     */
-    @Generated
-    @JsonCreator
-    private ChatChoice(@JsonProperty(value = "index") int index,
-        @JsonProperty(value = "finish_reason") CompletionsFinishReason finishReason) {
-        this.index = index;
-        this.finishReason = finishReason;
-    }
+    private ChatResponseMessage delta;
 
     /**
      * Get the message property: The chat message for a given chat completions prompt.
      *
      * @return the message value.
      */
     @Generated
-    public ChatMessage getMessage() {
+    public ChatResponseMessage getMessage() {
```

Comment:
Wondering do you know why `ChatResponseMessage` is a better name than the original name `ChatMessage`? And why not using `ChatResponseChoice` to replace `ChatChoice`

Reply:
Ah. There is a `ChatRequestMessage`.